### DISCUSSION TITLE
Gamemode Improvements: Ship Goals

### DISCUSSION THREAD

**Discussion by Absolutely-Steph** [BeeStation/NSV13#1147] [Ideas]
### Y'know station goals? How about **Ship Goals**?
Let me tell you about Ship Goals, currently the only "ship goal" is to finish a patrol, namely to defeat a certain number of enemies.
What if i tell you that It would be more fun to have different goals available? Maybe even ship specific ones?

**My Idea is:**
Every roundstart, the comms console spits out a piece of paper that states current mission parameters(Ship goal + modifiers?).

An example would be: "_Bring X amount of cargo Y to Czanek shipyards in System: VY Canis majoris_",
Possible modifiers could be a pirate band that wants your cargo(armada style), or that you don't have it yet and need to get it first, wether by trading or looting.

An example for ship specific missions would be mining missions for ships capable of it, like the eclipse, where you have to procure certain amounts of minerals and deliver it somewhere.

Possible Ship goals might include:

- Freight Transport (Bring stuff somewhere)
- Freight Procurement (Get stuff, bring it somewhere)
- Defeat Enemy Flagship (PVP)
- Mining (Get some amount of minerals, deliver it somewhere)
- Celestial Body Research (Fire probe torps into X amount of the type of body specified)
- Combat Patrol (Clear a specified area of enemies, maybe you get a list of systems adjacent to eachother)
- Route Mapping (Find a route to system X in brazil, maybe to a syndicate station or something)

Possible Modifiers would be:

- Pirate Band (Pirates are after something on your ship, they will pursue and try to board you)
- Privateer Band (Better equipped pirates, out to destroy you, or steal from you)
- Additional Cargo (Like freight transport, but less urgent)
- Infiltrators (Like traitors but more 007 agent like, they want to kill/steal/sabotage)
- FTL Anomaly (Higher chance for anomaly events when jumping, maybe reduced range)
- Recession (Prices in Stations are Higher, crew pay reduced?)
- Syndicate Uprising (Higher amounts of syndicate ships/fleets)

Any round has a slim chance to get 2 goals, and up to 2 Modifiers.
Maybe we even get a vote on roundend, what goal will be next?

**Comment by theboy6545** [comment]
> Infiltrators (Like traitors but more 007 agent like, they want to kill/steal/sabotage)

That's just a trator.

**Reply by Absolutely-Steph** [reply]
Except that they can happen even on extended, as they are part of the mission itself not a gamemode

**Reply by Omppusolttu** [reply]
I'd personally recommend making the crew vote on one of these objectives with only the modifier being random.
